controllers/ibmvpcmachine_controller.go

```diff
@@ -165,19 +166,32 @@ func (r *IBMVPCMachineReconciler) reconcileNormal(machineScope *scope.MachineSco
 		_, ok := machineScope.IBMVPCMachine.Labels[capiv1beta1.MachineControlPlaneLabelName]
```

Comment:
And we also need to handle the case of removing the node from control plane which is deletion part

Reply:
Can you please elaborate more on this scenario for my understanding.

Reply:
This scenario is:
- Deploys a cluster with 3 control plane replicas
- Scale the workers down 1, in this case 2 workers which gets deleted should also get removed from the backend pool.

Reply:
will work on adding the required code changes for the same. Thanks for pointing it out.

Reply:
Fixed, ptal.